lib/listcoef.gd

```diff
@@ -101,26 +101,53 @@ DeclareOperation(
 #O  MultRowVector( <list>, <mul> )
 ##
 ##  <#GAPDoc Label="MultRowVector">
+##  <#GAPDoc Label="MultRowVectorLeft">
+##  <#GAPDoc Label="MultRowVectorRight">
 ##  <ManSection>
 ##  <Oper Name="MultRowVector" Arg='list1, [poss1, list2, poss2, ]mul'/>
+##  <Oper Name="MultRowVectorLeft" Arg='list1, [poss1, list2, poss2, ]mul'/>
+##  <Oper Name="MultRowVectorRight" Arg='list1, [poss1, list2, poss2, ]mul'/>
+##  <Returns>nothing</Returns>
 ##
 ##  <Description>
-##  The five argument version of this operation replaces
+##  Note that <C>MultRowVector</C> is only a Synonym for
+##  <C>MultRowVectorLeft</C>.
+##  </P>
+##  These operations multiply <A>list1</A> with <A>mul</A> in-place and write
+##  the result to <A>list1</A>.
+##  <C>MultRowVectorLeft</C> multiplies <A>mul</A> to <A>list1</A> from the
+##  left. <C>MultRowVectorRight</C> does so from the right.
+##  The two-argument version simply multiplies each element of <A>list1</A>,
+##  in-place, by <A>mul</A> from the left.
```

Comment:
I was about to write "simply", but upon closer inspection, I'll rather say: Just drop that sentence, it is redundant.